KIPs/kip-1000.md

```diff
@@ -0,0 +1,138 @@
+---
+kip: 1000
```

Comment:
Please use the number of the issue. In this case, use 55.

Also, create an issue and make a link here like:

```
discussions-to: https://github.com/klaytn/kips/issues/37
```

Refer to https://github.com/klaytn/kips/pull/38/files#diff-9f77fd22a0b52c4fee3450bc89d6dd02bf757a0711e0d8ea1192950a4be108d7R5